test/test_data/test_validation.py

```diff
@@ -0,0 +1,91 @@
+import unittest
+
+from autosklearn.data.validation import InputValidator
+
+import numpy as np
+import pandas as pd
+
+
+class InputValidatorTest(unittest.TestCase):
+
+    def setUp(self):
+        self.X = [
+            [2.5, 3.3, 2, 5, 1, 1],
+            [1.0, 0.7, 1, 5, 1, 0],
+            [1.3, 0.8, 1, 4, 1, 1]
+        ]
+        self.y = [0, 1, 0]
+
+    def test_array_input(self):
```

Comment:
Could you please rename this test to `test_list_input` to avoid confusion with a numpy array?

Reply:
understood. Fixed in next commit.